integration_tests/CMakeLists.txt

```diff
@@ -514,7 +514,7 @@ RUN(NAME loop_03             LABELS cpython llvm llvm_jit c wasm wasm_x64)
 RUN(NAME loop_04             LABELS cpython llvm llvm_jit c)
 RUN(NAME loop_05             LABELS cpython llvm llvm_jit c)
 RUN(NAME loop_06             LABELS cpython llvm llvm_jit c NOFAST)
-RUN(NAME loop_07             LABELS cpython llvm llvm_jit c)
+RUN(NAME loop_07             LABELS cpython llvm llvm_jit)
```

Comment:
Since this doesn't work with the C backend anymore, can you please create a separate test for the new tests that you added?

Reply:
Yes, @tanay-man  please create a separate test for the new test.